receiver/adapter/accumulator/metrics.go

```diff
@@ -125,6 +126,9 @@ func populateDataPointsForHistogram(
 		h.SetTimestamp(timestamp)
 		d.ConvertToOtel(h)
 		addTagsToAttributes(h.Attributes(), tags)
+
+		// add special attribute to tell the exporter to handle this histogram datapoint differently
+		h.Attributes().PutStr(constants.AdapterReceiverAttribute, "")
```

Comment:
How do we ensure that this doesn't show up by accident if we export this metric using a different exporter (like the awsemfexporter)?